gdx/src/com/badlogic/gdx/maps/tiled/BaseTmjMapLoader.java

```diff
@@ -0,0 +1,746 @@
+						cell.setTile(tile);
+						layer.setCell(x, flipY ? height - 1 - y : y, cell);
+					}
+				}
+			}
+			JsonValue properties = element.get("properties");
+			if (properties != null) {
+				loadProperties(layer.getProperties(), properties);
+			}
+			parentLayers.add(layer);
+		}
+	}
+
+	protected void loadObjectGroup (TiledMap map, MapLayers parentLayers, JsonValue element) {
+		if (element.getString("type", "").equals("objectgroup")) {
+			MapLayer layer = new MapLayer();
+			loadBasicLayerInfo(layer, element);
+			JsonValue properties = element.get("properties");
+			if (properties != null) {
+				loadProperties(layer.getProperties(), properties);
+			}
+
+			for (JsonValue objectElement : element.get("objects")) {
+				loadObject(map, layer, objectElement);
+			}
+
+			parentLayers.add(layer);
+		}
+	}
+
+	protected void loadImageLayer (TiledMap map, MapLayers parentLayers, JsonValue element, FileHandle tmjFile,
+		ImageResolver imageResolver) {
+		if (element.getString("type", "").equals("imagelayer")) {
+			float x = element.getFloat("offsetx", 0);
+			float y = element.getFloat("offsety", 0);
+			if (flipY) y = mapHeightInPixels - y;
+
+			TextureRegion texture = null;
+
+			JsonValue image = element.get("image");
+
+			if (image != null) {
+				String source = image.getString("source");
+				FileHandle handle = getRelativeFileHandle(tmjFile, source);
+				texture = imageResolver.getImage(handle.path());
+				y -= texture.getRegionHeight();
+			}
+
+			TiledMapImageLayer layer = new TiledMapImageLayer(texture, x, y);
+
+			loadBasicLayerInfo(layer, element);
+
+			JsonValue properties = element.get("properties");
+			if (properties != null) {
+				loadProperties(layer.getProperties(), properties);
+			}
+
+			parentLayers.add(layer);
+		}
+	}
+
+	protected void loadBasicLayerInfo (MapLayer layer, JsonValue element) {
+		String name = element.getString("name");
+		float opacity = element.getFloat("opacity", 1.0f);
+		boolean visible = element.getBoolean("visible", true);
+		float offsetX = element.getFloat("offsetx", 0);
+		float offsetY = element.getFloat("offsety", 0);
+		float parallaxX = element.getFloat("parallaxx", 1f);
+		float parallaxY = element.getFloat("parallaxy", 1f);
+
+		layer.setName(name);
+		layer.setOpacity(opacity);
+		layer.setVisible(visible);
+		layer.setOffsetX(offsetX);
+		layer.setOffsetY(offsetY);
+		layer.setParallaxX(parallaxX);
+		layer.setParallaxY(parallaxY);
+	}
+
+	protected void loadObject (TiledMap map, MapLayer layer, JsonValue element) {
+		loadObject(map, layer.getObjects(), element, mapHeightInPixels);
+	}
+
+	protected void loadObject (TiledMap map, TiledMapTile tile, JsonValue element) {
+		loadObject(map, tile.getObjects(), element, tile.getTextureRegion().getRegionHeight());
+	}
+
+	protected void loadObject (TiledMap map, MapObjects objects, JsonValue element, float heightInPixels) {
+		if (element.getString("type", "").equals("object")) {
```

Comment:
This if statement needs to be removed. In the .tmj file there is no type ="object". All objects are stored in the json array of "objects". Each object individually has no type property as far as i can see. 
Because of this this entire loading process was skipped over and the Objects were never being loading.
To fix, Please **Remove** this: 
`if (element.getString("type", "").equals("object")) {`